File: cpp/conditional/SmallestNumber.cpp
/*
 * "Make a programm that recieves 3 integer values as inputs and then
 * prints which is the smallest number."
 *      -- Problem by Nelio Alves
 */

 #include<iostream>

 using namespace std;

 int main() {

    int first, second, third;

    cout << "Insert the first number: ";
    cin >> first;
    cout << "Insert the second number: ";
    cin >> second;
    cout << "Insert the third number: ";
    cin >> third;

    if ( (first < second) && (first < third) ) {
        cout <<  "\nSmallest number: " << first << endl;
    }
    else if ( second < third ) {
        cout << "\nSmallest number: " << second << endl;
    }
    else {
        cout << "\nSmmallest Number: " << third << endl;
    }

    return 0;
 }

File: cs/conditional/SmallestNumber.cs
/*
 * "Make a programm that recieves 3 integer values as inputs and then
 * prints which is the smallest number."
 *      -- Problem by Nelio Alves
 */

 using System;

 namespace SmallestNumber 
 {
    class SmallestNumber 
    {
        static void Main (string[] args) 
        {
            Console.Write("Insert the first number: ");
            int first = int.Parse(Console.ReadLine());
            Console.Write("Insert the second number: ");
            int second = int.Parse(Console.ReadLine());
            Console.Write("Insert rthe third number: ");
            int third = int.Parse(Console.ReadLine());
            
            int smallestNumber;
            if ( ( first < second ) && ( first < third ) )  {
                smallestNumber = first;
            } 
            else if ( second < third ) {
                smallestNumber = second;
            }
            else {
                smallestNumber = third;
            }

            Console.WriteLine("\nSmallest number: " + smallestNumber + "\n");
        }
    }
 }
